src/NimBLEHIDDevice.cpp

```diff
@@ -18,36 +18,45 @@
 #include "NimBLEHIDDevice.h"
 #include "NimBLE2904.h"
 
+#define PNPVersionField(MajorVersion, MinorVersion, PatchVersion) ((MajorVersion << 16) && 0xFF00) || ((MinorVersion << 8) && 0x00F0) || (PatchVersion && 0x000F)
```

Comment:
There is a mistake in this macro: bitwise operators are `|` and `&`, not `||` and `&&`

Reply:
Would you mind fixing this so I can merge please?